src/Infrastructure/Array/tests/makefile

```diff
@@ -7,6 +7,7 @@ run_uni:  run_unit_tests_uni
 LOCDIR	  = src/Infrastructure/Array/tests
 
 .NOTPARALLEL:
+#TESTS_BUILD = $(ESMF_TESTDIR)/ESMF_ArrayInfoUTest
```

Comment:
There is no ESMF_ArrayInfoUTest, and this looks like something left over from development that should not come in.